backend/app/notify/webhook.go

```diff
@@ -0,0 +1,142 @@
+package notify
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"strings"
+	"text/template"
+	"time"
+
+	log "github.com/go-pkgz/lgr"
+	"github.com/pkg/errors"
+)
+
+const (
+	webhookTimeOut  = 5 * time.Second
```

Comment:
nit: probably calling it "default" will clarify the intend better, i.e. webhookDefaultTimeOut

Reply:
Ok